test/test_mps.py

```diff
@@ -713,6 +712,7 @@ def mps_ops_modifier(ops):
         'special.xlog1py': [torch.bool, torch.int16, torch.int32, torch.int64, torch.uint8, torch.int8],
         'special.zeta': [torch.bool, torch.int16, torch.int32, torch.int64, torch.uint8, torch.int8],
         'special.chebyshev_polynomial_t': [torch.bool, torch.int16, torch.int32, torch.int64, torch.uint8, torch.int8],
+        'special.chebyshev_polynomial_u': [torch.bool, torch.int16, torch.int32, torch.int64, torch.uint8, torch.int8],
```

Comment:
did you miss the `v` and `w` variants?

Reply:
Nope, alas `u` and `w` are not tested by OpInfo as they've used to take a really long time, checking if this is still the case